src/com/oltpbenchmark/schema/Table.java

```diff
@@ -0,0 +1,23 @@
+package com.oltpbenchmark.schema;
+
+import java.util.List;
+
+public abstract class Table {
+    protected TableSchema schema = null;
+    
+    public Table (TableSchema schema) {
+        this.schema = schema;
+    }
+
+    // Return name of the table.
+    public String name() { 
+        return schema.name();
+    }
+
+    public String getDropDdl() {
+        return String.format("DROP TABLE IF EXISTS %s CASCADE", schema.name());
+    }
+    
+    // Return the DDL statements to create this table.
+    public abstract List<String> getCreateDdl();
```

Comment:
Is it necessary to use `List<String>` here? Is not it simper to use single string where multiple statements (if needed) are joined with ';' separator.

Reply:
I actually hadn't realized this. Thanks for pointing it out, made this change